gpkit/nomials/nomial_math.py

```diff
@@ -469,6 +451,25 @@ def __init__(self, left, oper, right):
         self.nomials = [self.left, self.right, self.p_lt, self.m_gt]
         self.nomials.extend(self._unsubbed)
 
+    def simplify_posy_ineq(self, exps, cs):
+        "Simplify a posy <= 1 by moving constants to the right side."
```

Comment:
Oh! Yeah, you're right, it should be private.
On Mar 25, 2016 20:23, "Warren Hoburg" <notifications@github.com> wrote:

> In gpkit/nomials/nomial_math.py
> <https://github.com/hoburg/gpkit/pull/604#discussion_r57504889>:
>
> > @@ -469,6 +451,25 @@ def __init__(self, left, oper, right):
> >          self.nomials = [self.left, self.right, self.p_lt, self.m_gt]
> >          self.nomials.extend(self._unsubbed)
> >
> > +    def simplify_posy_ineq(self, exps, cs):
> > +        "Simplify a posy <= 1 by moving constants to the right side."
>
> Agree with that.
>
> The confusion is that this method only makes sense when called with the
> exps and cs that arise in self._gen_unsubbed and self.as_posyslt1. maybe
> it should be private? If it's part of the user-facing API then I want to
> know what are valid inputs, and it doesn't seem to make sense to print
> self in the errors for arbitrary inputs.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/hoburg/gpkit/pull/604/files/f7179778993f51edd2c593d7c529d195644812ea#r57504889>
>